pkg/agent/handler/accessrequest.go

```diff
@@ -113,6 +119,37 @@ func (h *accessRequestHandler) Handle(action proto.Event_Type, meta *proto.Event
 	return err
 }
 
+func (h *accessRequestHandler) handleForTrace(action proto.Event_Type, meta *proto.EventMeta, resource *v1.ResourceInstance) error {
+	if resource.Kind != mv1.AccessRequestGVK().Kind {
+		return nil
+	}
+
+	if action == proto.Event_DELETED {
```

Comment:
Does the watch topic define delete events for AccessRequests?

Reply:
Already asked for that in Jason's PR

Reply:
Why do we need delete events for access requests when we handle it when the state is deleting with the finalizer?